libcnmc/cir_4_2015/F1.py

```diff
@@ -147,32 +147,15 @@ def consumer(self):
                             )
                             o_utmx = round(vertex['x'], 3)
                             o_utmy = round(vertex['y'], 3)
-                        if bloc_escomesa['node']:
```

Comment:
It's necessary to check if bloc_escomesa['node'] exists because when you doing bloc_escomesa['node'][0] or it will fail